t/venom/switches/common/check_wlan0.100_int_status.yml

```diff
@@ -0,0 +1,10 @@
+name: Check wlan0.2 interface status on wireless01
```

Comment:
Is this file still necessary ?

Reply:
No, you are right. I use a generic based on variable now.